### PR TITLE
CVE info for various Chrome releases

### PR DESCRIPTION
**Pull request by awhalley**

### CODE REVIEW

File: 2018/6xxx/CVE-2018-6169.json [hunk 1]
```diff
@@ -1,18 +1,63 @@
...
+                        "value": "Incorrect security UI"
+                    }
+                ]
+            }
+        ]
+    },
+    "references": {
+        "reference_data": [
+            {
+                "url": "https://crbug.com/394518"
```
Comment (cve-team) [inline 2018/6xxx/CVE-2018-6169.json]: Permission denied for this reference too.

File: 2017/15xxx/CVE-2017-15401.json [hunk 2]
```diff
@@ -1,18 +1,63 @@
...
+                {
+                    "vendor_name": "Google",
+                    "product": {
+                        "product_data": [
+                            {
+                                "product_name": "Chrome",
+                                "version": {
+                                    "version_data": [
+                                        {
+                                            "version_value": "62.0.3202.62",
```
[the remark this answers is not in the document]
Comment (awhalley) [reply 2017/15xxx/CVE-2017-15401.json]: Discrepancies such as these artifacts of our release process, and the fact we're so granular in our numbering. The first build on the 62 Stable branch that had this fixed was indeed 62.0.3202.62. There were a few more build made before the build that ended up being released as the first official Stable build, which incremented the build number.  There might be users that had version 62.0.3202.62, either though pre-stable rollout trials or via chromium embedders.

File: 2017/15xxx/CVE-2017-15420.json [hunk 3]
```diff
@@ -1,92 +1,93 @@
...
+                {
+                    "product": {
+                        "product_data": [
+                            {
+                                "product_name": "Chrome",
+                                "version": {
+                                    "version_data": [
+                                        {
+                                            "version_affected": "<",
+                                            "version_value": "63.0.3239.84"
```
[the remark this answers is not in the document]
Comment (awhalley) [reply 2017/15xxx/CVE-2017-15420.json]: The first build with that fix was indeed 63.0.3239.84

File: 2018/16xxx/CVE-2018-16072.json [hunk 4]
```diff
@@ -1,18 +1,63 @@
...
+                        "value": "Insufficient policy enforcement"
+                    }
+                ]
+            }
+        ]
+    },
+    "references": {
+        "reference_data": [
+            {
+                "url": "https://crbug.com/864283"
```
[the remark this answers is not in the document]
Comment (awhalley) [reply 2018/16xxx/CVE-2018-16072.json]: That one should have been, just opened it now.

File: 2018/16xxx/CVE-2018-16084.json [hunk 5]
```diff
@@ -1,18 +1,63 @@
...
+                        "value": "Inappropriate implementation"
+                    }
+                ]
+            }
+        ]
+    },
+    "references": {
+        "reference_data": [
+            {
+                "url": "https://crbug.com/865202"
```
[the remark this answers is not in the document]
Comment (awhalley) [reply 2018/16xxx/CVE-2018-16084.json]: This bug was embargoed, asked for confirmation it's OK to release.

File: 2018/17xxx/CVE-2018-17458.json [hunk 6]
```diff
@@ -1,18 +1,63 @@
...
+                        "value": "Inappropriate implementation"
+                    }
+                ]
+            }
+        ]
+    },
+    "references": {
+        "reference_data": [
+            {
+                "url": "https://crbug.com/875322"
```
[the remark this answers is not in the document]
Comment (awhalley) [reply 2018/17xxx/CVE-2018-17458.json]: Have asked if the view restriction is still needed.

File: 2018/17xxx/CVE-2018-17481.json [hunk 7]
```diff
@@ -1,83 +1,63 @@
...
-         }
-      ]
-   }
-}
+        ]
+    },
+    "references": {
+        "reference_data": [
+            {
+                "url": "https://crbug.com/901654"
```
[the remark this answers is not in the document]
Comment (awhalley) [reply 2018/17xxx/CVE-2018-17481.json]: Under Restrict-View-SecurityEmbargo, unlikely to be made public.

File: 2018/20xxx/CVE-2018-20065.json [hunk 8]
```diff
@@ -1,18 +1,63 @@
...
+                        "value": "Inappropriate implementation"
+                    }
+                ]
+            }
+        ]
+    },
+    "references": {
+        "reference_data": [
+            {
+                "url": "https://crbug.com/851821"
```
[the remark this answers is not in the document]
Comment (awhalley) [reply 2018/20xxx/CVE-2018-20065.json]: Due to be made public 14 weeks after fix, which is later this month.

File: 2018/20xxx/CVE-2018-20065.json [hunk 9]
```diff
@@ -1,18 +1,63 @@
...
+            },
+            {
+                "url": "https://chromereleases.googleblog.com/2018/12/stable-channel-update-for-desktop.html"
```
[the remark this answers is not in the document]
Comment (awhalley) [reply 2018/20xxx/CVE-2018-20065.json]: The CVE wasn't allocated when the release notes were published - the bug number was mentioned, but without the CVE. The reference has been updated.

File: 2018/20xxx/CVE-2018-20066.json [hunk 10]
```diff
@@ -1,18 +1,63 @@
...
+            }
+        ]
+    },
+    "references": {
+        "reference_data": [
+            {
+                "url": "https://crbug.com/856135"
+            },
+            {
+                "url": "https://chromereleases.googleblog.com/2018/12/stable-channel-update-for-desktop.html"
```
[the remark this answers is not in the document]
Comment (awhalley) [reply 2018/20xxx/CVE-2018-20066.json]: The CVE wasn't allocated when the release notes were published - the bug number was mentioned, but without the CVE. The reference has been updated.

File: 2018/20xxx/CVE-2018-20067.json [hunk 11]
```diff
@@ -1,18 +1,63 @@
...
+            }
+        ]
+    },
+    "references": {
+        "reference_data": [
+            {
+                "url": "https://crbug.com/879965"
+            },
+            {
+                "url": "https://chromereleases.googleblog.com/2018/12/stable-channel-update-for-desktop.html"
```
[the remark this answers is not in the document]
Comment (awhalley) [reply 2018/20xxx/CVE-2018-20067.json]: The CVE wasn't allocated when the release notes were published - the bug number was mentioned, but without the CVE. The reference has been updated.

File: 2018/20xxx/CVE-2018-20068.json [hunk 12]
```diff
@@ -1,18 +1,63 @@
...
+            }
+        ]
+    },
+    "references": {
+        "reference_data": [
+            {
+                "url": "https://crbug.com/882270"
+            },
+            {
+                "url": "https://chromereleases.googleblog.com/2018/12/stable-channel-update-for-desktop.html"
```
[the remark this answers is not in the document]
Comment (awhalley) [reply 2018/20xxx/CVE-2018-20068.json]: The CVE wasn't allocated when the release notes were published - the bug number was mentioned, but without the CVE. The reference has been updated.

File: 2018/20xxx/CVE-2018-20069.json [hunk 13]
```diff
@@ -1,18 +1,63 @@
...
+            }
+        ]
+    },
+    "references": {
+        "reference_data": [
+            {
+                "url": "https://crbug.com/890558"
+            },
+            {
+                "url": "https://chromereleases.googleblog.com/2018/12/stable-channel-update-for-desktop.html"
```
[the remark this answers is not in the document]
Comment (awhalley) [reply 2018/20xxx/CVE-2018-20069.json]: The CVE wasn't allocated when the release notes were published - the bug number was mentioned, but without the CVE. The reference has been updated.

File: 2018/20xxx/CVE-2018-20070.json [hunk 14]
```diff
@@ -1,18 +1,63 @@
...
+            }
+        ]
+    },
+    "references": {
+        "reference_data": [
+            {
+                "url": "https://crbug.com/895885"
+            },
+            {
+                "url": "https://chromereleases.googleblog.com/2018/12/stable-channel-update-for-desktop.html"
```
[the remark this answers is not in the document]
Comment (awhalley) [reply 2018/20xxx/CVE-2018-20070.json]: The CVE wasn't allocated when the release notes were published - the bug number was mentioned, but without the CVE. The reference has been updated.

File: 2017/15xxx/CVE-2017-15404.json [hunk 15]
```diff
@@ -1,18 +1,63 @@
...
+            {
+                "url": "https://chromereleases.googleblog.com/2017/10/stable-channel-updates-for-chrome-os.html"
+            }
+        ]
+    },
+    "description": {
+        "description_data": [
+            {
+                "lang": "eng",
+                "value": "An ability to process crash dumps under root privilegies and inappropriate symlinks handling could lead to a local privilege escalation in Crash Reporting in Google Chrome on Chrome OS prior to 61.0.3163.113 allowed a local attacker to perform privilege escalation via a crafted HTML page."
```
[the remark this answers is not in the document]
Comment (awhalley) [reply 2017/15xxx/CVE-2017-15404.json]: Thanks, will fix.

File: 2018/17xxx/CVE-2018-17458.json [hunk 16]
```diff
@@ -1,18 +1,63 @@
...
+            {
+                "url": "https://chromereleases.googleblog.com/2018/09/stable-channel-update-for-desktop_11.html"
+            }
+        ]
+    },
+    "description": {
+        "description_data": [
+            {
+                "lang": "eng",
+                "value": "An improper update of the WebAssembly diaptch table in WebAssembly in Google Chrome prior to 69.0.3497.92 allowed a remote attacker to execute arbitrary code inside a sandbox via a crafted HTML page."
```
[the remark this answers is not in the document]
Comment (awhalley) [reply 2018/17xxx/CVE-2018-17458.json]: Thanks, will fix.

File: 2018/20xxx/CVE-2018-20066.json [hunk 17]
```diff
@@ -1,18 +1,63 @@
...
+                        "value": "Use after free"
+                    }
+                ]
+            }
+        ]
+    },
+    "references": {
+        "reference_data": [
+            {
+                "url": "https://crbug.com/856135"
```
[the remark this answers is not in the document]
Comment (awhalley) [reply 2018/20xxx/CVE-2018-20066.json]: Due to be made public 14 weeks after fix, which is later this month.

File: 2018/20xxx/CVE-2018-20068.json [hunk 18]
```diff
@@ -1,18 +1,63 @@
...
+                        "value": "Inappropriate implementation"
+                    }
+                ]
+            }
+        ]
+    },
+    "references": {
+        "reference_data": [
+            {
+                "url": "https://crbug.com/882270"
```
[the remark this answers is not in the document]
Comment (awhalley) [reply 2018/20xxx/CVE-2018-20068.json]: Due to be made public 14 weeks after fix, which is today, so just opened it manually.

File: 2018/20xxx/CVE-2018-20069.json [hunk 19]
```diff
@@ -1,18 +1,63 @@
...
+                        "value": "Insufficient policy enforcement"
+                    }
+                ]
+            }
+        ]
+    },
+    "references": {
+        "reference_data": [
+            {
+                "url": "https://crbug.com/890558"
```
[the remark this answers is not in the document]
Comment (awhalley) [reply 2018/20xxx/CVE-2018-20069.json]: Due to be made public 14 weeks after fix, which is later this month.

File: 2018/20xxx/CVE-2018-20070.json [hunk 20]
```diff
@@ -1,18 +1,63 @@
...
+                        "value": "Insufficient policy enforcement"
+                    }
+                ]
+            }
+        ]
+    },
+    "references": {
+        "reference_data": [
+            {
+                "url": "https://crbug.com/895885"
```
[the remark this answers is not in the document]
Comment (awhalley) [reply 2018/20xxx/CVE-2018-20070.json]: Due to be made public 14 weeks after fix, which is later this month.

File: 2018/6xxx/CVE-2018-6091.json [hunk 21]
```diff
@@ -1,18 +1,63 @@
...
+                {
+                    "vendor_name": "Google",
+                    "product": {
+                        "product_data": [
+                            {
+                                "product_name": "Chrome",
+                                "version": {
+                                    "version_data": [
+                                        {
+                                            "version_value": "66.0.3359.106",
```
[the remark this answers is not in the document]
Comment (awhalley) [reply 2018/6xxx/CVE-2018-6091.json]: In this case there were additional fixes landed after 106, so 117 would be more accurate. Will fix.

File: 2018/6xxx/CVE-2018-6093.json [hunk 22]
```diff
@@ -1,18 +1,63 @@
...
+                {
+                    "vendor_name": "Google",
+                    "product": {
+                        "product_data": [
+                            {
+                                "product_name": "Chrome",
+                                "version": {
+                                    "version_data": [
+                                        {
+                                            "version_value": "66.0.3359.106",
```
[the remark this answers is not in the document]
Comment (awhalley) [reply 2018/6xxx/CVE-2018-6093.json]: There was an issue here, 66.0.3359.106 should have been 66.0.3359.117 for M66 bugs. Will fix.

**Comment by cve-team** [comment]
Thanks for your submission and for making the requested changes. These CVE IDs will be published on cve.mitre.org within 1-2 hours.

**Pull request merged**